### ISSUE TITLE
flashcanvas doesn't always draw embedded images

### ISSUE THREAD

**Issue by AMA3** [aFarkas/webshim#142]
There is a known bug in FlashCanvas where, due to a delay in loading images, they do not always appear on the canvas.  I've developed a very rudimentary fix for this, as well as for the problem (https://github.com/aFarkas/webshim/issues/135) of ExCanvas not drawing images in the correct place.  If anyone needs either, you can write some code similar to this.  It does not account for cropping:

``` javascript
var canvas = $('whatever selector gets your canvas element');
var context = canvas.getContext();
$('body').css('position','relative');
context.drawImage =
   function( img, x, y, w, h )
   {
      $('<img src="'+img.src+'"/>').appendTo('body').css( {
          height: h+'px',
          left: (x+canvas.offset().left)+'px',
          position: 'absolute',
          top: (y+canvas.offset().top)+'px',
          width: w+'px',
          zIndex: 1+canvas.css('zIndex') } )
   }

```

Basically, the code places an HTML img element on top of the canvas whenever drawImage is called.  Note that because the image appears over the canvas, it will hide any subsequent changes to the canvas at the image location.
